src/app/modules/ix-forms/components/ix-combobox/ix-combobox.harness.ts

```diff
@@ -42,6 +44,12 @@ export class IxComboboxHarness extends ComponentHarness implements IxFormControl
     await harness.selectOption({ text: optionLabel });
   }
 
+  async writeCustomValue(text: string): Promise<void> {
+    const input = await this.getMatInputHarness();
+    await input.setValue(text);
+    return (await input.host()).dispatchEvent('change');
```

Comment:
I don't see why we need to dispatch `change` here. It works without it, and ix-combobox doesn't seem to use it.